<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartyTypeDetailDialog</class>
 <widget class="QWidget" name="PartyTypeDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Party Type Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex"><number>0</number></property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title"><string>General</string></attribute>
      <layout class="QVBoxLayout" name="generalLayout">
       <item>
        <widget class="QGroupBox" name="basicInfoGroup">
         <property name="title">
          <string>Basic Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_codeEdit">
            <property name="text">
             <string>Code:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="codeEdit">
            <property name="placeholderText">
             <string>Enter party type code</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_nameEdit">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="placeholderText">
             <string>Enter display name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_descriptionEdit">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="descriptionEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter a description</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="provenanceTab">
      <attribute name="title"><string>Provenance</string></attribute>
      <layout class="QVBoxLayout" name="provenanceTabLayout">
       <item>
        <widget class="ores::qt::ProvenanceWidget" name="provenanceWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ores::qt::ProvenanceWidget</class>
   <extends>QWidget</extends>
   <header>ores.qt/ProvenanceWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
